airswap/airswap-marketplace
# reverting check for erc1155

# Description

`Fixes #85 `

Owned NFTs in past were still showing as owned. This bug was caused by an async handler in a filter method.

# Steps to test the fix

1. Set an erc721/erc1155 NFT collection address from which you owned an NFT in the past
2. Check that the NFT does not appear in the balances store

# Checklist:
